backend/test/app_configuration/test_update_configuration.py

```diff
@@ -170,3 +186,86 @@ def test_update_config_unexpected_exception(mock_app_config_dao):
 
     mock_app_config_dao.create.side_effect = ClientError(error_msg, "PutItem")
     assert lambda_handler(mock_event, mock_context) == expected_response
+
+
+@mock.patch.dict("os.environ", TEST_ENV_CONFIG, clear=True)
+@mock.patch("ml_space_lambda.app_configuration.policy_helper.notebook.iam")
+def test_update_instance_constraint_policies(iam):
```

Comment:
For testing organization, I think we should either handle this within the context of a valid app config update or move this test to a separate file I think.

I think if we are moving all of this to a separate file to de-clutter the app-config then it probably does make sense for us to move this to a separate test file too. If we are testing it as part of a call to update then I feel it makes more sense to keep here

Won't block on this. Choose what you think makes sense.

Reply:
I think splitting this out to its own test file probably makes more sense.